Lib/pathlib.py

```diff
@@ -1280,12 +1281,16 @@ def link_to(self, target):
 
     # Convenience functions for querying the stat results
 
-    def exists(self):
+    def exists(self, follow_symlinks=True):
         """
         Whether this path exists.
+
+        If path is a symlink: with `follow_symlinks=True`, return True if path
+        pointed to by the symlink exists; with follow_symlinks=False, return
+        True if the symlink itself exists.
```

Comment:
This seems unnecessary detail to me for a docstring. It's fairly self-explanatory I think.

Reply:
I was a bit concerned about users who are not familiar with dangling symlinks, in that case this param would be confusing without explanation. I think I was either unfamiliar with them or haven't seen them in too long of a time when I was working on this issue. If you feel most users of this method should be familiar with them or be able to look it up, I will remove the note, just wanted to point out that was my reason for adding it.

Reply:
Makes sense. Maybe a small wording tweak, then?

> This method normally follows symlinks; to check whether a symlink exists, add the argument ``follow_symlinks=False``.